src/obstacle/MeshObstacle.cxx

```diff
@@ -58,32 +59,32 @@ MeshObstacle::MeshObstacle()
 }
 
 
-static void cfvec3ListToArray(const std::vector<cfvec3>& list,
+static void cfvec3ListToArray(const std::vector<glm::vec3>& list,
```

Comment:
this is mis-named, since it's not a `cfvec3` converter anymore

Reply:
renamed